metabase/metabase
# Only run BE tests on static viz FE changes

https://github.com/metabase/metabase/issues/33615

### Description
- Updates the Static Viz webpack config to output a yaml file of sources included in the bundle
- Removes `frontend_all` as part of the `backend_all` file path
- During the Files Changed step of the BE workflow, static viz bundle is built and we check if any modified files are in the generated yaml file.

### How to verify
https://github.com/metabase/metabase/pull/33752 and https://github.com/metabase/metabase/pull/33750 were tests to check behavior.
